CHANGELOG.md

```diff
@@ -15,3 +15,9 @@ First release.
 - Templates: MacVLAN launch-xrd example, xr-compose template
 - Sample xr-compose topologies: 'simple-bgp', 'bgp-ospf-triangle, 'segment-routing'
 - Tests: host-check UT
+
+
+### v1.0.1 (2022-09-06)
```

Comment:
Should probably update the date to the day of the release (tomorrow?), but not too important :)